### PR TITLE
adding custom-auth-lambda-triggers repo

### PR DESCRIPTION
**Pull request by MattToczek**

### CODE REVIEW

File: custom-auth-lambda-triggers.tf [hunk 1]
```diff
@@ -0,0 +1,35 @@
+resource "github_repository" "custom-auth-lambda-triggers" {
+  name        = "custom-auth-lambda-triggers"
```
Comment (dan-hill2802) [inline custom-auth-lambda-triggers.tf:2]: this name doesn't seem correct. Are these lambdas specific to the analytical environment?

Comment (MattToczek) [reply custom-auth-lambda-triggers.tf:2]: They are indeed - I will change it to be more specific

File: dataworks-analytical-custom-auth-flow.tf [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  allow_merge_commit = false
+  has_issues         = true
+
+  lifecycle {
+    prevent_destroy = true
+  }
+}
+
+resource "github_team_repository" "example_dataworks" {
```
Comment (dan-hill2802) [inline dataworks-analytical-custom-auth-flow.tf:14]: resource name needs changing

File: dataworks-analytical-custom-auth-flow.tf [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  repository = "${github_repository.dataworks-analytical-custom-auth-flow.name}"
+  team_id    = "${github_team.dataworks.id}"
+  permission = "push"
+}
+
+resource "github_branch_protection" "example_master" {
```
Comment (dan-hill2802) [inline dataworks-analytical-custom-auth-flow.tf:20]: resource name needs changing

**Pull request merged**